ansible_ai_connect_chatbot/README.md

```diff
@@ -0,0 +1,69 @@
+# Ansible AI Connect Chatbot
+
+## Build from source
+
+To start building the "Admin Portal" project, you're going to need:
+
+- node (`18.20.0` was used for development)
+- npm (`10.5.0` was used for development)
+
+For installing dependencies, **make sure that the current directory
+is the `ansible_ai_connect_chatbot` sub-directory first**:
+```commandline
+ $ pwd
```

Comment:
`pwd` prints the present working directory.  Did you mean `cd`?

Reply:
I meant that user could verify the current directory with pwd, but cd would more straightforward. I'll change it to
```
cd (your_git_work_directory)/ansible-ai-connect-service/ansible_ai_connect_chatbot
```